src/query/constraint/src/problem.rs

```diff
@@ -57,3 +58,30 @@ pub fn assert_int_is_not_null(
     solver.pop(1);
     result
 }
+
+/// The same as `assert_int_is_not_null`, but for multiple variables.
+pub fn assert_int_is_not_null_multiple_variables(
```

Comment:
so we can remove assert_int_is_not_null because assert_int_is_not_null_multiple_variables covers the use case.

Reply:
and rename it to assert_all_ints_are_not_null